viewer.py

```diff
@@ -606,12 +716,15 @@ def parseOntology(root, parent=-1):
     return ont
 
 
-def writeFile(data, file, **kwds):
+def writeFile(data, file):
     dataDir = os.path.dirname(file)
     if dataDir != '' and not os.path.exists(dataDir):
         os.makedirs(dataDir)
-    data.write(file, **kwds)
 
+    if max(data.shape) > 200 and min(data.shape) > 200:
```

Comment:
Added this to avoid errors when creating .ma files. Since lower resolution nrrd files are smaller than (200,200,200), I was getting out of bounds errors. Specially using ccf files with 100 and 50um resolutions. Maybe I should just get rid of the 'chunks=(200,200,200)' in writeFile(data, file) function